### DISCUSSION TITLE
CVE-2022-1304 vulnerability in docker image

### DISCUSSION THREAD

**Discussion by p-l-l** [dotnet/dotnet-docker#3726] [General]
Recently detected this [vulnerability](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-1304) on image `mcr.microsoft.com/dotnet/aspnet:6.0`

Introduced by:
e2fsprogs/libss2@1.46.2-2
e2fsprogs/libext2fs2@1.46.2-2
krb5/libgssapi-krb5-2@1.18.3-6+deb11u1

Just wanted to let you know, thanks for any feedback if there are plans on fixing this

**Comment by mthalman** [comment]
It doesn't look like there's even a fix available at this time for Debian. From https://security-tracker.debian.org/tracker/CVE-2022-1304:

![image](https://user-images.githubusercontent.com/15789599/164717143-ec4dc2a7-d605-4e81-a96a-24d9f259ff29.png)

Even if there was a fix available, it's the base image, `debian:bullseye-slim`, that contains this package. So we need to wait until it's available in that image. Once it's fixed there, it'll naturally flow to the .NET images that depend on that base image as part of our auto-rebuilding workflow.
